tools/DARIAH-DE-Geo-Browser-CSV.json

```diff
@@ -0,0 +1,36 @@
+{
+    "task": "Visualisation",
+    "deployment": "development",
+    "name": "DARIAH-DE Geo-Browser (CSV)",
+    "logo": "dariah.svg",
```

Comment:
The file name does not match the .svg (it's `dariah-de` and not `dariah`)

Reply:
thanks, changed!